src/main/java/uk/gov/hmcts/cmc/claimstore/services/documentmanagement/Classification.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.cmc.claimstore.services.documentmanagement;
+
+public enum Classification {
```

Comment:
This belongs in the document management client imo

Reply:
Agree on that.

Reply:
Done